var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -54,3 +57,5 @@ def configure_args(self):
             '--enable-pc-files',
             '--with-pkg-config-libdir={0}/lib/pkgconfig'.format(self.prefix)
         ]
+        if '+symlinks' in self.spec:
+            opts += ["--enable-symlinks"]
```

Comment:
You are right. The return was missing and the ```configure(*opts)``` was red.

 I really shouldn't comment without explicit testing... Or while being hungry. Sorry for that